app/src/main/res/values/dimens.xml

```diff
@@ -4,4 +4,5 @@
     <dimen name="margin_40">40dp</dimen>
     <dimen name="margin_80">80dp</dimen>
     <dimen name="default_line_height">26dp</dimen>
+    <dimen name="max_elevation">16dp</dimen>
```

Comment:
I'd keep a very large number (say 1000) to make clear that this is to bring to front and not styling related. Maybe rename to reflect it better.